ionoscloud/resource_nic.go

```diff
@@ -261,6 +282,22 @@ func getNicData(d *schema.ResourceData, path string) ionoscloud.Nic {
 		}
 	}
 
+	if v, ok := d.GetOk(path + "ipv6_ips"); ok {
+		raw := v.([]interface{})
+		if raw != nil && len(raw) > 0 {
+			ipv6_ips := make([]string, 0)
```

Comment:
curious, cna you try using utils.DecodeInterfaceToStruct(raw, ipv6_ips)  instead of the for